packages/splitter/src/SplitterContainer.tsx

```diff
@@ -30,19 +31,26 @@ export const SplitterContainer: React.FC<ISplitterContainerProps> = props => {
 
 SplitterContainer.propTypes = {
   ariaLabel: PropTypes.string,
-  primaryPaneId: PropTypes.string.isRequired,
   children: PropTypes.func,
   render: PropTypes.func,
   environment: PropTypes.oneOfType([
-    PropTypes.instanceOf(Document),
+    PropTypes.instanceOf(Window),
```

Comment:
It seems like `Window` would be undefined in server side environments. I ran the `useSplitter` hook through Next.js and got:

<img width="1023" alt="Screen Shot 2021-12-16 at 8 56 28 AM" src="https://user-images.githubusercontent.com/1811365/146415437-e9385aef-f47d-46ec-9c0b-a75c0795447f.png">

Reply:
That's a great catch. 👍

Reply:
I removed the `window` type check in https://github.com/zendeskgarden/react-containers/pull/405/commits/1fcd3fe466056a77363624600aee432203f87351